packages/server-admin-ui/src/views/ServerConfig/BasicProvider.js

```diff
@@ -316,6 +316,37 @@ class RemoveNullsInput extends Component {
   }
 }
 
+class AppendChecksum extends Component {
+  constructor(props) {
+    super(props)
+    this.props.value.appendChecksum =
```

Comment:
What’s this for?

Reply:
Sorry, I'm not too much familiar with this stuff. 
I used https://github.com/SignalK/signalk-server-node/blob/master/packages/server-admin-ui/src/views/ServerConfig/BasicProvider.js#L290:L294 as reference. 

I'll will remove it from my code... should it be removed from L290-L294 as well ?